### ISSUE TITLE
Ongoing maintenance of project

### ISSUE THREAD

**Issue by HaleTom** [willghatch/zsh-hooks#7]
Hi William,

I'm wanting to make two small changes:

* Turn `$f` into a local variable (created via the code `for f in ...`).
* Update install instructions (zplugin as been renamed to zinit)

I note that RobertAudi has made many good updates:
https://github.com/willghatch/zsh-hooks/compare/master...RobertAudi:master

But that that repository is archived.

Would you be willing to accept a PR of the RobertAudi changes, then a PR of my changes?

Or would you be willing to transfer this repository into an organisation, eg `zsh-hooks/zsh-hooks`?  Then I could make these changes myself, and hopefully enlist others to continue your great work.

If it assists, I could create a `zsh-hooks` organisation and give you repository creation permissions in the receiving organisation, and guide you through the transfer process.

[There are several advantages of transferring over forking](https://docs.github.com/en/github/administering-a-repository/transferring-a-repository#whats-transferred-with-a-repository).

**Comment by willghatch** [comment]
Yes, I'm absolutely open to PRs.  I am also happy to transfer it to an organization, since I don't see myself really maintaining this going forward.  Go ahead and make the organization and I'll transfer it.

After transferring, does the old url with my username still redirect to it?

**Comment by HaleTom** [comment]
Cheers!

Yup, including `git` operations, too :)

> All links to the previous repository location are automatically redirected to the new location. When you use git clone, git fetch, or git push on a transferred repository, these commands will redirect to the new repository location or URL.

**Comment by HaleTom** [comment]
@willghatch I've sent you an invitaiton to the `zsh-hooks` organisation, with Owner permissions required to enact:

https://docs.github.com/en/github/administering-a-repository/transferring-a-repository#transferring-a-repository-owned-by-your-user-account

Let me know if you hit any roadblocks and I'll see if I can help you out!

**Comment by willghatch** [comment]
Done.  Good luck with further maintenance of the package!

**Comment by HaleTom** [comment]
Cheers.  I'll do my best to look after it and make it a community effort.

Before you likely disappear... Do you have any leads / hunches on the 
history hook not working?

**Comment by willghatch** [comment]
On Tue, Sep 15, 2020 at 10:20:35PM -0700, Tom Hale wrote:
>Cheers.  I'll do my best to look after it and make it a community effort.
>
>Before you likely disappear...

I'll still follow the repo and get emails about issues and such, but
probably more as a lurker than an active participant.

>Do you have any leads / hunches on the
>history hook not working?

No.  I don't think I personally ever had a good use for the history
hook, so I don't think I ever looked into it very deeply.  (Or if I
did I don't remember.)  That said, I wrote the comment about the
double free years ago.  It could be that newer versions of zsh have
already fixed this bug.

**Comment by HaleTom** [comment]
Cheers for the reply and the lurking watchful eyes.

I'll push some commits in the next few days.
